horizons/ai/aiplayer/productionbuilder.py

```diff
@@ -263,24 +263,24 @@ def display(self):
 			return
 
 		tile_colors = {
-			BUILDING_PURPOSE.ROAD:            ( 30,  30,  30),
```

Comment:
Let's just keep the old layout instead of messing with stuff like that. I believe we got enough opinions from the team to support that decision.